specification/ai/data-plane/Face/preview/v1.1-preview.1/Face.json

```diff
@@ -0,0 +1,7948 @@
+            "schema": {
+              "$ref": "#/definitions/CompareResult"
+            }
+          },
+          "default": {
+            "description": "An unexpected error response.",
+            "schema": {
+              "$ref": "#/definitions/Azure.Core.Foundations.ErrorResponse"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "type": "string",
+                "description": "String error code indicating what went wrong."
+              }
+            }
+          }
+        }
+      }
+    },
+    "/face/{apiVersion}/detect": {
+      "post": {
+        "operationId": "DetectFromUrl",
+        "description": "A remote procedure call (RPC) operation.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionPathParameter"
+          },
+          {
+            "$ref": "#/parameters/FaceDetectionOptions.returnFaceId"
+          },
+          {
+            "$ref": "#/parameters/FaceDetectionOptions.returnFaceLandmarks"
+          },
+          {
+            "$ref": "#/parameters/FaceDetectionOptions.returnFaceAttributes"
+          },
+          {
+            "$ref": "#/parameters/FaceDetectionOptions.recognitionModel"
+          },
+          {
+            "$ref": "#/parameters/FaceDetectionOptions.returnRecognitionModel"
+          },
+          {
+            "$ref": "#/parameters/FaceDetectionOptions.detectionModel"
+          },
+          {
+            "$ref": "#/parameters/FaceDetectionOptions.faceIdTimeToLive"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "url": {
+                  "type": "string",
+                  "description": "URL of input image."
+                }
+              },
+              "required": [
+                "url"
+              ]
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The request has succeeded.",
+            "schema": {
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/FaceDetectionResult"
+              },
+              "x-ms-identifiers": []
+            }
+          },
+          "default": {
+            "description": "An unexpected error response.",
+            "schema": {
+              "$ref": "#/definitions/Azure.Core.Foundations.ErrorResponse"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "type": "string",
+                "description": "String error code indicating what went wrong."
+              }
+            }
+          }
+        }
+      }
+    },
+    "/face/{apiVersion}/detectLiveness/singleModal/sessions": {
+      "get": {
+        "operationId": "LivenessSessions_ListSessions",
+        "description": "Operation that lists resources in a paginated way.",
```

Comment:
How does pagination work for this API?

Reply:
It's the description that comes from the TypeSpec Azure.Core operation. Our list operation only support query by top and start, I think it's not fulfilling the pagination of Azure API. I'll override the description.